camp_fin/management/commands/import_transactions.py

```diff
@@ -129,7 +210,12 @@ def import_expenditures(self, f, year):
                         received_date__year=year,
                     ).delete()
 
-                self.make_contribution(record, None, filing).save()
+                contribution = self.make_contribution(record, None, filing)
+                batch.append(contribution)
+
+                if not len(batch) % batch_size:
```

Comment:
when doing modulo, i think it's better to have the form that

```
len(batch) % batch_size == 0
```

i think it's just a touch more explicit.

Reply:
Love it. Done.